ext/geos_c_impl/polygon.c

```diff
@@ -253,12 +253,6 @@ cmethod_create(VALUE module,
   linear_ring_type = rgeo_feature_linear_ring_module;
   exterior_geom = rgeo_convert_to_detached_geos_geometry(
     exterior, factory, linear_ring_type, NULL, &state);
-  if (state) {
-    rb_exc_raise(rb_errinfo());
-  }
-  if (!exterior_geom) {
-    return Qnil;
-  }
```

Comment:
Again, I think this is safe to remove because `rgeo_convert_to_detached_geos_geometry` checks for both of these, but I'm not quite sure.